docker/compose.deploy.yaml

```
@@ -0,0 +1,267 @@
+      - source: redis_data
+        target: /data
+        type: volume
+    profiles:
+      - redis
+
+volumes:
+  caddy_config:
+  caddy_data:
+  minio_data:
+  postgres_data:
+  redis_data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)